docs/Advanced-Usage/Debugging-and-Profiling-on-FPGA/AutoCounter.rst

```diff
@@ -95,19 +105,41 @@ By default, the read-rate is set to 0 cycles, which disables AutoCounter.
    readrate=100
 
 
-Upon setting this value, when you run a workload, an AutoCounter output file
-will be placed in the ``sim_slot_<slot #>`` directory on the F1 instance under
-the name ``AUTOCOUNTERFILE<N>``, with one file generated per clock domain
-containing an AutoCounter event. The header of each output file indicates the
-associated clock domain and its frequency relative to the base clock.
-
 .. Note:: AutoCounter is designed as a coarse-grained observability mechanism, as sampling 
       each counter requires two (blocking) MMIO reads (each read takes O(100) ns on EC2 F1).
       As a result sampling at intervals less than O(10000) cycles may adversely affect
       simulation performance for large numbers of counters.
-      If you intend on reading counters at a finer granularity, please consider using
+      If you intend on reading counters at a finer granularity, consider using
       synthesizable printfs.
 
+AutoCounter CSV Output Format
+---------------------------------
+AutoCounter output files are CSVs generated in the working directory where the
+simulator was invoked (this applies to metasimulators too), with the default
+names ``AUTOCOUNTERFILE<N>``, one per clock domain. The CSV output format is
```

Comment:
Shouldn't the name have `.csv` attached at the end?

Reply:
I can append a csv, but I wasn't sure that was what people wanted. 

I'd be down if we renamed this plus arg: https://github.com/firesim/firesim/blob/master/sim/midas/src/main/cc/bridges/autocounter.cc#L54
to +autocounter-base-filename?

Reply:
I think appending `.csv` makes it extremely clear how to read the file and makes it nice if people start opening the file in other tools. I'm fine with renaming the plusarg.